### PR TITLE
Add public methods to access install and uninstall sql files

### PR DESCRIPTION
**Pull request by fimac**
Adds in the public methods to call from the adapter to install the sql script files.

### CODE REVIEW

**Review by freshtonic — APPROVED** [review]
Nice

File: cipherstash-pg.gemspec [hunk 1]
```diff
@@ -23,12 +23,14 @@ Gem::Specification.new do |spec|
...
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features|vendor)/}) }
   end
   spec.extensions    = ["ext/extconf.rb"]
   spec.require_paths = ["lib"]
   spec.cert_chain    = ["certs/ged.pem"]
   spec.rdoc_options  = ["--main", "README.rdoc",
                         "--title", "PG: The Ruby PostgreSQL Driver"]
   spec.extra_rdoc_files = `git ls-files -z *.rdoc lib/*.rb lib/*/*.rb ext/*.c ext/*.h`.split("\x0")
+  spec.files << "vendor/driver/database-extensions/postgresql/install.sql"
+  spec.files << "vendor/driver/database-extensions/postgresql/uninstall.sql"
```
Comment (damncabbage) [inline cipherstash-pg.gemspec:35]: This path isn't valid; there's nothing in `vendor/driver`.

(The submodule isn't present.)

**Review by damncabbage — CHANGES_REQUESTED** [review]

File: cipherstash-pg.gemspec (hunk 1 above)
Comment (fimac) [reply cipherstash-pg.gemspec:35]: @damncabbage oh, hmm, it's present when I view it locally? 

Screenshot 2023-02-21 at 2 01 24 pm

Comment (fimac) [reply cipherstash-pg.gemspec:35]: @damncabbage I've fixed that up, the submodule has been added in the PR.

**Review by damncabbage — APPROVED** [review]

**Pull request merged**